blueocean-dashboard/src/main/js/components/PullRequest.jsx

```diff
@@ -52,7 +53,10 @@ export default class PullRequest extends Component {
             <td>{title || '-'}</td>
             <td>{author || '-'}</td>
             <td><ReadableDate date={endTime} liveUpdate /></td>
-            <td><RunPipeline organization={organization} pipeline={pipelineName} branch={name} /></td>
+            <td>
+                <RunPipeline organization={organization} pipeline={pipelineName} branch={name} />
+                <Extensions.Renderer extensionPoint="jenkins.pipeline.pullrequests.list.action" />
```

Comment:
I was trying to be consistent with some of the other extension point names:
* jenkins.pipeline.list.top   (area above the main pipelines list)
* jenkins.pipeline.run.result   (from run details)

The other extension point we currently have in "jenkins.test.result" from the test page.